src/tinyformat.h

```diff
@@ -179,13 +179,19 @@ namespace tfm = tinyformat;
 
 namespace tinyformat {
 
+// Similar to std::runtime_format from C++26.
```

Comment:
nit:
```diff
+// Added for Bitcoin Core.
```
Maybe use some comment style to group the modifications to avoid repeating this, similar to https://www.doxygen.nl/manual/grouping.html